ext/pdo_oci/oci_driver.c

```diff
@@ -539,6 +539,29 @@ static int oci_handle_set_attribute(pdo_dbh_t *dbh, zend_long attr, zval *val) /
 #else
 			oci_drv_error("Unsupported attribute type");
 			return 0;
+#endif
+		}
+		case PDO_OCI_ATTR_CALL_TIMEOUT:
+		{
+#if (OCI_MAJOR_VERSION >= 18)
+			if (lval < 0) {
+				return 1;
+			} else if (lval > UB4MAXVAL) {
+				return 1;
```

Comment:
These checks need to be changed need to warn, throw exceptions, or just return false...